.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "abieos"]
```

Comment:
Now that we are including git submodules we should probably add information to the README explaining the extra steps needed to clone the project.